crypto/crypto/src/multi_ed25519.rs

```diff
@@ -430,7 +430,10 @@ impl TryFrom<&[u8]> for MultiEd25519Signature {
             return Err(CryptoMaterialError::WrongLengthError);
         }
 
-        let bitmap = bytes[length - BITMAP_NUM_OF_BYTES..].try_into().unwrap();
```

Comment:
that was a weird unwrap! Can it be re-written as a simple .into()? Or maybe just by copying into an array of the right size?

Reply:
` let bitmap = bytes[length - BITMAP_NUM_OF_BYTES..].into();
`
results into `the trait `std::convert::From<&[u8]>` is not implemented for `[u8; 4]``

Reply:
I think returning a DeserializationError is most appropriate as the caller already expects this one in some cases.

Reply:
looked a bit more at it, yeah I agree that it's the correct thing to do. BTW that function could be re-written shorter as:

```rust
fn try_from(bytes: &[u8]) -> std::result::Result<MultiEd25519Signature, CryptoMaterialError> {
      let bytes = bytes
        .chunks_exact(ED25519_SIGNATURE_LENGTH);

        // check correct number of signatures
        if bytes.len() < 0 || bytes.len() > MAX_NUM_OF_KEYS {
          return Err(CryptoMaterialError::WrongLengthError);
        }

        // get bitmap
        let bitmap: [u8; BITMAP_NUM_OF_BYTES] = bytes.remainder().try_into().map_err(Err(CryptoMaterialError::WrongLengthError))?;
        if bitmap_count_ones(bitmap) != num_of_sigs as u32 {
          return Err(CryptoMaterialError::DeserializationError);
        }

        // get signatures
        let signatures: Result<Vec<Ed25519Signature>, _> = bytes
          .map(Ed25519Signature::try_from)
          .collect()?;
        
        // return multi-sig
        Ok(MultiEd25519Signature { signatures, bitmap })
    }
```